exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/sklearn/sklearn_train_telescope.ipynb

```diff
@@ -91,7 +91,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    model.fit(X_train.values, y_train)
</pre><p>Why? The model is happy to take a DataFrame as an input.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/316/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
We observed an error message as specified in [comment-2293085752](https://github.com/exasol/ai-lab/pull/316#issuecomment-2293085752).
The notebook [sklearn_train_abalone.ipynb](https://github.com/exasol/ai-lab/blob/main/exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/sklearn/sklearn_train_abalone.ipynb?short_path=0c7d1ba#L152) did use `.values` already.

Reply:
<p>Hmm, that means the model has lost the feature names during serialization/deserialization. This may have something to do with the version mismatch, which we are fixing here. It's worth in investigating.</p>